src/main/kotlin/com/katanox/tabour/integration/sqs/core/consumer/SqsEventHandlerRegistry.kt

```diff
@@ -27,46 +29,29 @@ class SqsEventHandlerRegistry(
         val pollers: MutableSet<SqsEventPoller> = HashSet()
         for (registration in registrations) {
             pollers.add(createPollerForHandler(registration))
-            logger.info("initialized SqsMessagePoller '{}'", registration.javaClass.canonicalName)
+            logger.info("initialized SqsMessagePoller ${registration.javaClass.canonicalName}")
         }
         return pollers
     }
 
     private fun createPollerForHandler(registration: SqsEventHandler): SqsEventPoller {
         return SqsEventPoller(
-            name = registration.javaClass.canonicalName,
+            queueUrl = registration.sqsQueueUrl,
             eventHandler = registration,
-            eventFetcher = createFetcherForHandler(registration),
-            pollerThreadPool = createPollingThreadPool(registration),
-            handlerThreadPool = createHandlerThreadPool(registration),
-            pollingProperties = eventPollerProperties,
-            sqsConfiguration = sqsConfiguration,
-            exceptionHandler = ExceptionHandler.defaultExceptionHandler()
-        )
-    }
-
-    private fun createFetcherForHandler(registration: SqsEventHandler): SqsEventFetcher {
-        return SqsEventFetcher(registration.sqsQueueUrl, sqsConfiguration, eventPollerProperties)
-    }
-
-    private fun createPollingThreadPool(registration: SqsEventHandler): ScheduledThreadPoolExecutor {
-        return ThreadPools.blockingScheduledThreadPool(
-            eventPollerProperties.pollingThreads,
-            String.format("%s-poller", registration.javaClass.canonicalName)
-        )
-    }
-
-    private fun createHandlerThreadPool(registration: SqsEventHandler): ThreadPoolExecutor {
-        return ThreadPools.blockingThreadPool(
-            eventHandlerProperties.threadPoolSize,
-            eventHandlerProperties.queueSize,
-            String.format("%s-handler", registration.javaClass.canonicalName)
+            client = sqsConfiguration.amazonSQSAsync(),
+            tabourConfigs = tabourAutoConfigs,
+            pollerConfigs = pollerConfigs
         )
     }
 
+    @OptIn(ExperimentalCoroutinesApi::class)
     fun start() {
         for (poller in pollers) {
-            poller.start()
+            GlobalScope.launch {
```

Comment:
i would remove this and move the coroutine stuff to the class itself.  For example a poller may not use coroutines, which makes this redudant.